### PR TITLE
feat: Add option to build `arm64` binary

### LINKED ISSUE #248: [Enhancement] Ship `arm64` and `x64` binaries for macOS
### Feature summary

As part of #52, we wanted to ship a universal build for the recorder. There are a few reasons that preclude us from achieving this, so for the time being we will ship the recorder with separate Intel/Apple silicon binaries.

As a side benefit, these separate binaries will be significantly smaller than the universal build by a factor of close to 100%.

### Why is this feature important?

The recorder runs faster and has a snappier UI experience when build for native arm systems, so by providing this option out of the box users of newer Apple machines will have a nicer time using the recorder.

### Linked resources

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by justinkambic**
## Summary

Resolves #248.

Causes the `build`/`release` scripts to generate an Apple silicon binary in addition to the Intel one.

## Implementation details

Simply updates the build config for the recorder.

## How to validate this change

You can do an `npm run pack` command to build fresh binaries locally. You should be able to run both the `x64` and the `arm64` versions without issues.

### CODE REVIEW

**Comment by kyungeunni** [comment]
@justinkambic After our call, I found a way to include the correct browser directory per build. It should download and bundle the correct browser per os/architecture including Apple Silicon.

File: electron-builder.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ afterPack: "./scripts/after-pack.js"
 files:
   - build/**/*
   - scripts/**/*
+extraResources:
```
Comment (justinkambic) [inline electron-builder.yml:8]: Wow great work this is much cleaner!

File: scripts/download-chromium.js [hunk 2]
```diff
@@ -80,15 +85,15 @@ function translatePlatform(platform) {
   }
 }
 
-exports.downloadForPlatform = async function downloadForPlatform(platform) {
-  platform = translatePlatform(platform);
+exports.downloadForPlatform = async function downloadForPlatform(electron_platform, arch) {
```
Comment (justinkambic) [inline scripts/download-chromium.js:88]: Nice name update.

**Comment by kyungeunni** [comment]
Addressed the suggested artifact naming improvements!! Now all the artifacts have the same name format including the arch. 
image

**Comment by kyungeunni** [comment]
Hey @justinkambic I think this is ready to be reviewed, then merged if you are happy with it!

**Review by kyungeunni — APPROVED** [review]

**Pull request merged**